crates/l2/contracts/src/l1/OnChainProposer.sol

```diff
@@ -13,9 +13,15 @@ import {IPicoVerifier} from "./interfaces/IPicoVerifier.sol";
 /// @title OnChainProposer contract.
 /// @author LambdaClass
 contract OnChainProposer is IOnChainProposer, ReentrancyGuard {
+    /// @notice Committed blocks data.
+    /// @dev This struct holds the information about the committed blocks.
+    /// @dev processedDepositLogsRoot is the Merkle root of the logs of the
+    /// deposits that were processed in the block being committed. The amount of
+    /// logs that is encoded in this root are to be removed from the
+    /// pendingDepositLogs queue of the CommonBridge contract.
     struct BlockCommitmentInfo {
-        bytes32 commitmentHash;
-        bytes32 depositLogs;
+        bytes32 stateDiffKZGVersionedHash;
+        bytes32 processedDepositLogsRoot;
```

Comment:
I'd change `processedDepositLogsRoot` to `processedDepositLogsHash` here (or maybe even `processedDepositLogsRollingHash`)

Reply:
It's rollinga then.

Reply:
Done in [eef8cb5](https://github.com/lambdaclass/ethrex/pull/2551/commits/eef8cb55c298f53f0736f15c3a6560f966e5c321).